src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -537,4 +537,23 @@ Use the dropdown to view and switch to other projects this file may belong to.</
   <data name="Package_uninstall_failed_0" xml:space="preserve">
     <value>Package uninstall failed: {0}</value>
   </data>
+  <data name="IntellisenseBuildFailedMessage" xml:space="preserve">
+    <value>Error encountered while loading the project. Some project features, such as full solution analysis for the failed project and projects that depend on it, have been disabled.</value>
+  </data>
+  <data name="IntellisenseBuildFailedTitle" xml:space="preserve">
+    <value>Project loading failed.</value>
+  </data>
+  <data name="IntellisenseBuildFailedDescription" xml:space="preserve">
+    <value>To see what caused the issue, please try below.
+
+1. Close VS
+2. Set environment variable (set TraceDesignTime=true)
```

Comment:
Reorder 2 and 3, perhaps, and then this can be "open a Visual Studio Developer Command Prompt, and then run set TraceDesignTime=true"? Then it makes 4 easier to explain.